.gitignore

```diff
@@ -58,5 +58,5 @@ env[23]/
 # [sass] Files generated by Sass
 *.css
 
-static/js/modules/
```

Comment:
Is `statis/js/modules` no longer used? If so, great! :+1:

Reply:
Yup, removed the folder.